src/module/exports.rs

```diff
@@ -62,8 +68,17 @@ impl ModuleExports {
         &mut self.arena[id]
     }
 
+    fn get_safe(&self, id: ExportId) -> Option<&Export> {
+        self.arena.get(id)
+    }
+
+    fn get_mut_safe(&mut self, id: ExportId) -> Option<&mut Export> {
```

Comment:
"safe" is a misnomer here. For now, I think we should just access the arenas directly in the public methods below.

Longer term, we should make the regular `get` variants return options and not panic.

Reply:
Count me in for this refactoring. I wish it was a mono-repo :)